update.sh

```diff
@@ -9,25 +9,58 @@ if [ ${#versions[@]} -eq 0 ]; then
 fi
 versions=( "${versions[@]%/}" )
 
-packagesBase='http://apt.postgresql.org/pub/repos/apt/dists/jessie-pgdg'
-mainList="$(curl -fsSL "$packagesBase/main/binary-amd64/Packages.bz2" | bunzip2)"
+declare -A debianSuite=(
+	[9.2]='jessie'
+	[9.3]='jessie'
+	[9.4]='jessie'
+	[9.5]='jessie'
+	[9.6]='jessie'
+	[10]='stretch'
+)
+declare -A alpineVersion=(
+	[9.2]='3.5'
+	[9.3]='3.5'
+	[9.4]='3.5'
+	[9.5]='3.5'
+	[9.6]='3.5'
+	[10]='3.6'
+)
 
 # https://www.mirrorservice.org/sites/ftp.ossp.org/pkg/lib/uuid/?C=M;O=D
 osspUuidVersion='1.6.2'
 osspUuidHash='11a615225baa5f8bb686824423f50e4427acd3f70d394765bdff32801f0fd5b0'
 
+lastSuite=
 travisEnv=
 for version in "${versions[@]}"; do
+	if [ "${debianSuite[$version]}" != "$lastSuite" ]; then
+		lastSuite="${debianSuite[$version]}"
+		packagesBase='http://apt.postgresql.org/pub/repos/apt/dists/'"${debianSuite[$version]}"'-pgdg'
+		mainList="$(curl -fsSL "$packagesBase/main/binary-amd64/Packages.bz2" | bunzip2)"
+	fi
+
 	versionList="$(echo "$mainList"; curl -fsSL "$packagesBase/$version/binary-amd64/Packages.bz2" | bunzip2)"
 	fullVersion="$(echo "$versionList" | awk -F ': ' '$1 == "Package" { pkg = $2 } $1 == "Version" && pkg == "postgresql-'"$version"'" { print $2; exit }' || true)"
 	(
 		set -x
 		cp docker-entrypoint.sh "$version/"
-		sed 's/%%PG_MAJOR%%/'"$version"'/g; s/%%PG_VERSION%%/'"$fullVersion"'/g' Dockerfile-debian.template > "$version/Dockerfile"
+		sed -e 's/%%PG_MAJOR%%/'"$version"'/g;' \
+			-e 's/%%PG_VERSION%%/'"$fullVersion"'/g' \
+			-e 's/%%DEBIAN_SUITE%%/'"${debianSuite[$version]}"'/g' \
+			Dockerfile-debian.template > "$version/Dockerfile"
+		if [ "$version" = '10' ]; then
```

Comment:
I wasn't sure if it was a permanent change for 10 to provide contrib or if that was possibly only a change in the beta version. Doing it this way we get the failure on 10.1 when the package still does not exist instead of silently not installing it.